src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Deletes tags from all students that contain tags that match the given string.
+ */
+public class DeleteTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteTag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Given a String, removes every tag from every student"
+            + " that matches the String\n"
+            + "Example: " + COMMAND_WORD + "Olympiad Student";
```

Comment:
Missing a space character between COMMAND__WORD and "Olympiad Student". :)

Reply:
Fixed